### DISCUSSION TITLE
In the display of publications, how can I directly show all the authors?

### DISCUSSION THREAD

**Discussion by ShuhongDai** [alshedivat/al-folio#2518] [Q&A] [answered]
Great job on this! As someone who is completely new to frontend development, I now have my own personal homepage! 

However, I still have some concerns that I hope to resolve. In the display of publications, how can I directly show all the authors? As shown in the image, the current abbreviation like "4 more authors" might lead to an incomplete display of the author list,
<img width="588" alt="截屏2024-06-19 03 04 44" src="https://github.com/alshedivat/al-folio/assets/79570336/9f26b3bc-11f1-4e79-8bc6-bfacdb66866d">
 and I would like to list all the authors' names directly. How can I modify the code to do this? 
<img width="638" alt="截屏2024-06-19 03 05 09" src="https://github.com/alshedivat/al-folio/assets/79570336/5a96ab50-9868-47f5-a9a6-cac553e57066">
Additionally, how can I make a specific author's name bold?

**Comment by george-gca** [comment, marked as answer]
To show more authors, you can change the number here:

https://github.com/alshedivat/al-folio/blob/3b1c10844f62db235dded4f7e5d8d520414143b5/_config.yml#L359

As to make authors's name bold, why do you want to do this? Just checking because maybe something like what is implemented in #2512 suits your need more.

**Reply by ShuhongDai** [reply]
Thank you for your prompt response. All of my issues have been resolved through the elegant solutions you provided!
